code/modules/projectiles/guns/smgs.dm

```diff
@@ -111,12 +111,13 @@
 	aim_speed_modifier = 0.55
 
 	accuracy_mult = 1.15
-	accuracy_mult_unwielded = 0.9
+	accuracy_mult_unwielded = 0.6
```

Comment:
accuracy is very much RNG, we generally prefer using scatter for balancing, since 99% of the cases guns always hit anyway

Reply:
See splinter, up the akimbo stats.